### DISCUSSION TITLE
Font error

### DISCUSSION THREAD

**Discussion by augustocdias** [neovide/neovide#2262] [Q&A] [answered]
I'm getting this error when opening neovide. Then it freezes after some time.

![image](https://github.com/neovide/neovide/assets/1080789/4090f8d9-024a-4e50-857a-492fd41a09cf)

My font config is: `vim.o.guifont = 'FiraCode Nerd Font,codicons:h12:#e-subpixelantialias'`

**Comment by fredizzimo** [comment, marked as answer]
We should probably improve the error message...

It looks like the `codeicons` font is not found, I'm not sure if such a font even exist.

The hang should be fixed on main if you compile from source.

**Reply by augustocdias** [reply]
You're right. It's a typo there. It should be `codicon`, but it does exist: https://microsoft.github.io/vscode-codicons/dist/codicon.html
